### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by themayor897** [blu3mania/npp-papyrus#10] [General]
Just wanted to say thank you for updating this. I had an updated version on NexusMods that someone else had made, but I have hidden it and redirected people here. Excited to use it!

**Comment by blu3mania** [comment]
> Just wanted to say thank you for updating this. I had an updated version on NexusMods that someone else had made, but I have hidden it and redirected people here. Excited to use it!

Thanks for the kind words!

**Comment by BeranabusBarnes** [comment]
Just found this, and it's a great help!
